docs/source/_themes/sphinx_rtd_theme/layout.html

```diff
@@ -17,10 +17,11 @@
   <title>{{ title|striptags|e }}{{ titlesuffix }}</title>
   {% endblock %}
 
-  <link rel="shortcut icon" href="{{ pathto('_static/favicon.ico', 1) }}?v4"/>
+  <link rel="shortcut icon" href="{{ pathto('_static/favicon.ico', 1) }}?brocade"/>
```

Comment:
why is this here? afaik the whole _themes for brocade is coming from ipfabric-docs, https://github.com/StackStorm/ipfabric-docs/tree/master/docs/source/_themes/sphinx_rtd_theme